.flowconfig

```diff
@@ -4,7 +4,7 @@ build
 [include]
 
 [libs]
-flow.js
+flowTypes.js
```

Comment:
renamed to flowTypes.js because if we have `flow.js` and I do `flow` in terminal it opens the file in my textEditor instead checking types. Actually seems the name should be <modulename>.js.flow , but with our project name it looks ugly: `brewskey.js-api.js.flow`